one-pager-maker/src/pages/ResetPassword.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { useCallback, useState } from 'react';
+import {
+    Box,
+    Button,
+    TextField,
+} from '@mui/material';
+import { sendPasswordResetEmail } from 'firebase/auth';
+import { auth } from '../firebase';
+import { useNavigate } from 'react-router-dom';
+
+const ResetPassword = () => {
+
+    const navigate = useNavigate();
+
+    const [formData, setFormData] = useState({ email: "" });
+
+    const handleFormData = useCallback((event: React.ChangeEvent<HTMLTextAreaElement | HTMLInputElement>) => {
+        setFormData(formData => ({
+            ...formData,
+            [event.target.id]: event.target.value,
+        }))
+    }, [])
+
+
+    const reset = useCallback(() => {
+        // TODO: redirect sign in page 
+        sendPasswordResetEmail(
+            auth,
+            formData.email,
+        ).then(() => {
+            alert('send mail and please check your mail box')
```

Comment:
This message below might be better than yours?
"We sent an email to you. Please check your mail box."